pkg/artifacts/artifact_storage_test.go

```diff
@@ -155,6 +155,9 @@ func TestInitializeArtifactStorageWithConfigMap(t *testing.T) {
 			if err != nil {
 				t.Fatalf("Somehow had error initializing artifact storage run out of fake client: %s", err)
 			}
+			if err := CleanupArtifactStorage(c.pipelinerun, fakekubeclient, logger); err != nil {
```

Comment:
could we have separate unit test cases specifically for `CleanupArtifactStorage` instead of adding this call to the existing tests?

Reply:
Sure? Though it feels to me like this is the right place for it - otherwise we're just going to have tests that also call `InitializeArtifactStorage` before calling `CleanupArtifactStorage` anyway.

Reply:
Done.